Tests/PubNubContractTest/Steps/Objects/PubNubObjectsChannelMetadataContractTestSteps.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  PubNubObjectsSpaceContractTestSteps.swift
+//
+//  PubNub Real-time Cloud-Hosted Push API and Push Notification Client Frameworks
+//  Copyright © 2022 PubNub Inc.
+//  https://www.pubnub.com/
+//  https://www.pubnub.com/terms
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Cucumberish
+import Foundation
+import PubNub
+import XCTest
+
+public class PubNubObjectsChannelMetadataContractTestSteps: PubNubContractTestCase {
+  var channelMetadata: PubNubChannelMetadata?
+  
+  public override func handleBeforeHook() {
+    channelMetadata = nil
+    super.handleBeforeHook()
+  }
+  
+  public override func setup() {
+    startCucumberHookEventsListening()
+    
+    Given("^the id for '(.*)' channel$") { args, _ in
+      guard args?.count == 1, let channelName = args?.first?.lowercased() else {
```

Comment:
This is ok, but you can also use the `XCTUnwrap` assert if you would like to:

https://developer.apple.com/documentation/xctest/3380195-xctunwrap
https://www.hackingwithswift.com/example-code/testing/how-to-check-and-unwrap-optionals-in-tests-using-xctunwrap

Reply:
@jguz-pubnub I've pushed changes for this.

Reply:
@jguz-pubnub had to revert. This won't work properly with `C` functions (`Given` is `C` function defined in `Cucumberish`).